mlrun/runtimes/base.py

```diff
@@ -427,6 +431,14 @@ def _get_cmd_args(self, runobj: RunObject):
             raise ValueError('cannot use "pass" mode with handler')
 
         if code:
+            if self.kind == "databricks-job":
+                code = b64decode(code).decode("utf-8")
+                code = (
+                    code
+                    + mlrun.runtimes.get_runtime_class(self.kind).get_enriched_code()
+                )
```

Comment:
The idea was that `get_enriched_code()` will be defined in `BaseRuntime` and will always get called, but in most cases will do nothing.
And I think you don't want to do `code + get_enriched_code()` since in some cases the runtime may want to modify or add stuff at the beginning.
So, the basic implementation will be:
```
get_enriched_code(code: str) -> str:
    return code
```
And it's up to the `get_enriched_code` to actually do the base64 decode/encode, to save the decoding/encoding in the general case.